modules/order/order.resolver.js

```diff
@@ -0,0 +1,60 @@
+const ordersService = require('./order.service');
+const { ORDER_NOT_FOUND } = require('../../error-messages/orders.messages');
+
+const ordersQuery = {
+  getOrderById: async (parent, args) => {
+    const order = await ordersService.getOrderById(args.id);
+    if (order) {
+      return order;
+    }
+    return {
+      statusCode: 404,
+      message: ORDER_NOT_FOUND,
+    };
+  },
+  getAllOrders: async (parent, args) => {
+    try {
+      return await ordersService.getAllOrders(args);
+    } catch (e) {
```

Comment:
Why do we need this `catch` block if `ordersService.getAllOrders(...)` cannot throw a 404 exceptions?

Reply:
fixed